clients/nft_bridge/main.ts

```diff
@@ -1,6 +1,6 @@
 import yargs from "yargs";
 
-const { hideBin } = require('yargs/helpers')
+import { hideBin } from 'yargs/helpers'
```

Comment:
So tl;dr, the javascript module system is like 10 competing standards, and it is a mess.  There are a bunch of different ways of packaging and resolving modules (commonjs, esModules, etc), each with their own way of importing, doing introspection, etc.

The way you bring a module into your javascript project, require vs import, determines what code is actually pulled in. Due to the state of modules being a mess, most popular packages will export the same source code as multiple module types, to make it easy for users to consume it.

In this case, if you `const { hideBin } = require('yargs/helpers')`, you will only resolve the commonjs module, which has no type definitions. However, if you `import { hideBin } from 'yargs/helpers'`, then you will also resolve `@types/yargs`, which will supply typings for the library, allow the typescript compiler to do tree-shaking and lazy-loading (if applicable), and other general goodness.

For Typescript projects, if you can `import` it rather than `require` it, you pretty much always should. If you'd like to read about it, it's a hot mess all the way down, and it gets worse when you try to support both the browser and nodejs runtimes. [https://www.typescriptlang.org/docs/handbook/module-resolution.html](https://www.typescriptlang.org/docs/handbook/module-resolution.html)

Reply:
types from require:
![require-ts](https://user-images.githubusercontent.com/2636098/162538342-29a6e86f-98ed-48dc-b178-5f70c4688245.png)

types from import:
![import-ts](https://user-images.githubusercontent.com/2636098/162538385-260bdec5-f2d8-40cd-adab-139cfe529d07.png)

This is low hanging fruit - don't throw away types, use what is supplied by the module.